app/views/boats/new.html.erb

```diff
@@ -2,11 +2,11 @@
   <div class="">
     <%= simple_form_for (@boat) do |f| %>
       <div class="form-group mt-5">
-        <%= f.input :title %>
-        <%= f.input :description %>
-        <%= f.input :address, id: "address" %>
-        <%= f.input :category, collection: Boat::CATEGORIES %>
-        <%= f.input :price %>
+        <%= f.input :title, :placeholder => 'Describe your rental' %>
+        <%= f.input :description, :placeholder => 'My boat is answome and measure 50 feets ...', :label => "What can you tell us about you boat ?"  %>
```

Comment:
My boat is answome and measure 50 feets ...
==> My boat is answome and is 50 feet long...